volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -1710,3 +1710,18 @@ def get_available_pages(self) -> List:
                     )
 
         return vacb_list
+
+
+class LDR_DATA_TABLE_ENTRY(objects.StructType):
+    def get_load_count(self) -> Optional[int]:
+        try:
+            LoadCount = self.LoadCount
+        except Exception:
+            try:
+                LoadCount = self.ObsoleteLoadCount
+            except Exception:
+                LoadCount = None
+        if LoadCount == 65535:
+            LoadCount = -1
```

Comment:
Is this a valid situation, or should this also return `None` as a means of error?

Reply:
If it's supposed to be signed, you could `.cast('CORRECT_TYPE')` it to the right type?

Reply:
A cast is what I was thinking, thanks for the hint. I casted to a short and now it shows -1 like it should.

Not sure what you meant about return None as means of an error. I don't think there should be a case where both LoadCount and ObsoleteLoadCount don't exist, but just had the exception there to prevent a crash in case that ever happened.